### PR TITLE
Hide <textarea> When Toggling Preview in Legacy Editor

### PR DESCRIPTION
**Pull request by noi5e**
Fix for this bug mentioned in this comment: https://github.com/publiclab/plots2/pull/9221#issuecomment-782782858

---
(This issue is part of the larger Comment Editor Overhaul Project with Outreachy. Refer to Planning Issue #9069 for more context)

### CODE REVIEW

File: app/assets/javascripts/editor.js [hunk 1]
```diff
@@ -3,12 +3,13 @@ class Editor {
   //   defaultForm - when the Editor is initialized, there needs to be a default editor form:
   //     1. the main comment form in multi-comment wikis, questions, & research notes.
   //     2. the only editor form on /wiki/new and /wiki/edit
-  //   elementIDPrefixes - the ID naming conventions are different depending on context:
+  //   isSingleFormPage - to distinguish between a) pages with multiple comments b) pages like /wiki/new, and /features/new with only one comment form
+  //     elements have different ID naming conventions on the two kinds of pages:
   //     1. multi-form pages with multiple comments: #comment-preview-123
   //     2. /wiki/new and /wiki/edit: #preview-main
-  constructor(defaultForm = "main", elementIDPrefixes = {}) {
+  constructor(defaultForm = "main", isSingleFormPage = false) {
```
Comment (noi5e) [inline app/assets/javascripts/editor.js:10]: This is hopefully a lot less confusing than `elementIDPrefixes`.

**Review by jywarren — APPROVED** [review]
Looks great, do you think it's worth adding a test for the various places this is used?

**Comment by noi5e** [comment]
> Looks great, do you think it's worth adding a test for the various places this is used?

I'm not sure, I don't really have a good sense for that. I can if you think it's worth it. `/wiki/new` and `/wiki/edit` probably get the most traffic out of all the possibilities right?

**Comment by jywarren** [comment]
I guess i think it would be worthwhile esp if we already have a test set up
for those views and can just add an extra "preview pressing" step? That way
this won't recur ever.

Thanks, Will!!!

**Comment by noi5e** [comment]
Added a new test for `/wiki/new` preview. This one's ready to merge!

**Comment by jywarren** [comment]
Nice, thank you @noi5e !!!

**Pull request merged**